test/lint/lint-submodule.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+export LC_ALL=C
+EXIT_CODE=0
+
+CMD=$(find . -type f | grep .gitmodules)
```

Comment:
What if this is executed in test folder?

Reply:
Then it will only look for .gitmodules in the test folder. Is there a way to go to the root dir?